CppSamples/DisplayInformation/ShowPopup/ShowPopup.h

```diff
@@ -31,18 +31,19 @@ class GeoElement;
 class IdentifyLayerResult;
 class Map;
 class MapQuickView;
-class PopupManager;
+class Popup;
 }
 
 Q_MOC_INCLUDE("MapQuickView.h")
 Q_MOC_INCLUDE("PopupManager.h")
+Q_MOC_INCLUDE("Popup.h")
 
 class ShowPopup : public QObject
 {
   Q_OBJECT
 
   Q_PROPERTY(Esri::ArcGISRuntime::MapQuickView* mapView READ mapView WRITE setMapView NOTIFY mapViewChanged)
-  Q_PROPERTY(QQmlListProperty<Esri::ArcGISRuntime::PopupManager> popupManagers READ popupManagers NOTIFY popupManagersChanged)
+  Q_PROPERTY(Esri::ArcGISRuntime::Popup* popup READ popup NOTIFY popupChanged)
```

Comment:
Consider using `MEMBER` here. It avoids the need to add a trivial getter function.

https://doc.qt.io/qt-6.8/properties.html

Optional.